src/java/org/apache/cassandra/cql3/functions/OperationFcts.java

```diff
@@ -154,6 +164,25 @@ protected ByteBuffer executeOnTemporals(TemporalType<?> type,
             throw new UnsupportedOperationException();
         }
 
+        /**
```

Comment:
If we add these it would be good it were std javadoc and I would add it to both methods.